announcements/2020-Week-31/aws.md

```diff
@@ -0,0 +1,20 @@
+# :zap: This Week in AWS Releases (July 31, 2020):zap:
+
+[AWS v12.0.0](https://github.com/giantswarm/releases/tree/master/aws/v12.0.0) brings Kubernetes v1.17.
+
+[AWS v9.3.6](https://github.com/giantswarm/releases/blob/master/aws/v9.3.6) updates managed apps to the latest releases.
```

Comment:
```suggestion
[AWS v9.3.6](https://github.com/giantswarm/releases/tree/master/aws/v9.3.6) updates managed apps to the latest releases.
```

Reply:
damn this is not merged yet. thanks for catching